README.md

```diff
@@ -1,37 +1,43 @@
 # The​ ​task
 
- Many​ ​of​ ​us​ ​are​ ​avid​ ​foosball​ ​players,​ ​but​ ​running​ ​around​ ​the​ ​office​ ​to​ ​form​ ​teams​ ​is​ ​both​ ​tiring and​ ​wastes​ ​lots​ ​of​ ​our​ ​precious​ ​time.​ ​Therefore,​ ​we​ ​would​ ​appreciate​ ​a​ ​bit​ ​of​ ​help​ ​with​ ​this.​ ​This task​ ​is​ ​about​ ​making​ ​a​ ​tool​ ​that​ ​would​ ​enable​ ​us​ ​to​ ​organise​ ​players​ ​and​ ​maybe​ ​even​ ​track some​ ​statistics​ ​ ​-​ ​essentially,​ ​getting​ ​a​ ​game​ ​started​ ​without​ ​moving​ ​our​ ​bottoms​ ​for​ ​our​ ​comfy chairs.​ ​Who​ ​knows,​ ​if​ ​things​ ​go​ ​well​ ​you​ ​might​ ​get​ ​to​ ​use​ ​it​ ​too.
+ Many​ ​of​ ​us​ ​are​ ​avid​ ​foosball​ ​players,​ ​but​ ​running​ ​around​ ​the​ ​office​ ​to​ ​form​ ​teams​ ​is​ ​both​ ​tiring and​ ​wastes​ ​lots​ ​of​ ​our​ ​precious​ ​time.​ ​Therefore,​ ​we​ ​would​ ​appreciate​ ​a​ ​bit​ ​of​ ​help​ ​with​ ​this.​ ​This task​ ​is​ ​about​ ​making​ ​a​ ​tool​ ​that​ ​would​ ​enable​ ​us​ ​to​ ​organise​ ​players​ ​and​ ​maybe​ ​even​ ​track some​ ​statistics​ ​ ​-​ ​essentially,​ ​getting​ ​a​ ​game​ ​started​ ​without​ ​moving​ ​our​ ​bottoms​ ​for​ ​our​ ​comfy chairs.​ ​Who​ ​knows,​ ​if​ ​things​ ​go​ ​well​ ​you​ ​might​ ​get​ ​to​ ​use​ ​it​ ​too!
 
- The​ ​app​ ​has​ ​to​ ​allow​ ​us​ ​to​ ​access​ ​a​ ​lobby,​ ​view​ ​players’​ ​details,​ ​create​ ​a​ ​game​ ​where​ ​you​ ​invite others​ ​as​ ​well​ ​as​ ​rearrange​ ​teams​ ​and​ ​then,​ ​after​ ​a​ ​match,​ ​one​ ​can​ ​add​ ​some​ ​stats,​ ​which​ ​will be​ ​stored​ ​for​ ​later.
+ At a minimum, the​ ​app​ should ​allow​ ​us​ ​to​ ​access​ ​a​ ​lobby and​ ​create​ ​a​ ​game​ ​where​ ​you​ ​invite others​. If you want to and have enough time you can add bonus features: ​players’​ ​details view, ​rearranging ​teams​ or,​ ​after​ ​a​ ​match,​ ​add​ing and storing ​some​ ​stats.
 
- We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ ​If​ ​you​ ​don’t​ ​manage​ ​to​ ​implement everything,​ ​no​ ​worries​ ​-​ ​we’d​ ​be​ ​very​ ​pleasantly​ ​surprised​ ​if​ ​you​ ​did.
+ We​ ​ask​ ​you​ ​not​ ​to​ ​spend​ ​​more​ ​than​ ​4​ ​hours​​ ​for​ ​this​ ​task.​ Don't worry if ​you​ ​don’t​ ​manage​ ​to​ ​implement a gazillion of features or breathtaking design -​ focus on the create game and lobby functionalities. We'll appreciate clean elegant code, a thoughtful solution and a nice project structure!
  
  # The​ ​functionality
  - Lobby​ ​(with​ ​a​ ​list​ ​of​ ​players​ ​and​ ​some​ ​information) 
- - Detailed​ ​player​ ​view
  - Create​ ​a​ ​game 
     - Invite​ ​players
-    - Rearrange​ ​teams​ ​by​ ​reassigning​ ​players 
-    - Validation​ ​(teams​ ​can​ ​only​ ​consist​ ​of​ ​4​ ​or​ ​2​ ​players)
+
+
+## Nice to haves
```

Comment:
I would try to use wording that emphesis that it's not mandatory. Ania ofcourse I will follow your lead here. which one is the softest:
- nice to have
- optional
- not mandatory

Reply:
Let's try `optional`.